README.md

```diff
@@ -144,32 +151,38 @@ Play with trading simulators where you can engage with the market and practice y
  * [Coinmarketcap](https://coinmarketcap.com/) - CoinMarketCap provides information about all digital currencies that are traded in at least one public exchange.
  * [Coingecko](https://www.coingecko.com/en) - Like CMC, CoinGecko is a platform for tracking cryptocurrencies and blockchain assets.
  * [Coin360](https://coin360.com/) - Heatmap of cryptocurrency prices, market capitalizations, and volumes.
- 
+ * [Crypto Compare](https://www.cryptocompare.com/) - Like CMC, Crypto Compare is a platform for tracking cryptocurrencies and blockchain assets.
+ * [Crypto Hedgehog](https://cryptohedgehog.io/) - Crypto Hedgehog aggregates data from CoinMarketCap, Coin Gecko, Crypto Compare, Messari, Santiment, etc. Research includes market data, overviews, teams, partners, ICO information, feeds, and more.
+ * [Messari](https://messari.io/) - Messari provides reliable data and market intelligence for crypto investors and professionals.
+
+
+
 ## Portfolio and Tax Trackers
 
  * [BlockFolio](https://blockfolio.com/) - Mobile cryptocurrency portfolio tracker app.
  * [CoinTracking](https://cointracking.info/) - CoinTracking is a portfolio tracking platform with tax report generation tool.
+ * [Crypto Hedgehog](https://cryptohedgehog.io/) - Crypto Hedgehog is a portfolio management platform where you can track your balances, build asset classes, set strategies, and trade through any connected exchange.
```

Comment:
Crypto Hedgehog seems more like a market research/tracker tool and I think it fits more to the "Market Trackers" section only.

Reply:
So Crypto Hedgehog has an incredibly extensive portfolio / wallet / trading areas that are accessible once you login. It could fit into either category, but I'd encourage you to try it out.

...I might also make a differentiation between Portfolio Trackers and Tax Trackers. Taxes are a pretty specific subset of apps, and alot of them don't handle portfolio stuff. (I.e. Bitcoin.tax, Blox.io, CryptoTrader.tax). 

The big portfolio trackers are BlockFolio and CoinTracking, but could also include Delta, CoinStats, ShapeShift, and Crypto Compare (though CC's portfolio tools are...not the best)

**Possible Next Steps:**
- Include or Exclude Crypto Hedgehog from the list
- Split list into Portfolio Trackers and Tax Trackers
- Add Bitcoin.tax, Blox.io, CryptoTrader.tax to Tax Trackers?
- Add Delta, CoinStats, and Crypto Hedgehog to Portfolio Trackers

Let me know your thoughts!

Reply:
I tried the Crypto Hedgehog by openning an account, I think it does not fit into "Market Trackers" section, I am thinking maybe I should rename "Market Screeners" category to "Market Research" or "Market Screening and Research" and include Crypto Hedgehog into it? What do you think?